DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/AllDelegates/Index.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model AllDelegatesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/allDelegates.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Delegates";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Delegates/All";

```

Comment:
Ah, the HeaderPath should be the centre dashboard in all the tracking system pages. Ideally we'd only specify the Application on ViewData and then get HeaderPathName & HeaderPath based on that, but that's tech debt.. 😢